code/controllers/configuration.dm

```diff
@@ -12,7 +12,7 @@
 	return ..()
 
 /datum/configuration/vv_edit_var(var_name, var_value)
-	var/static/list/banned_edits = list("cross_address", "cross_allowed", "autoadmin", "autoadmin_rank")
+	var/static/list/banned_edits = list("cross_address", "cross_allowed", "autoadmin", "autoadmin_rank", "invoke_youtubedl")

```

Comment:
Is there anywhere else I need to place this to prevent tampering?

Reply:
no